coordinator/statement_executor.go

```diff
@@ -394,6 +385,12 @@ func (e *StatementExecutor) executeSetPasswordUserStatement(q *influxql.SetPassw
 }
 
 func (e *StatementExecutor) executeSelectStatement(stmt *influxql.SelectStatement, ctx *influxql.ExecutionContext) error {
+	// If there is a target and we are in a read only context, exit early since
+	// we will return an error.
+	if stmt.Target != nil && ctx.ReadOnly {
```

Comment:
That's what I'm saying though.  It should not throw an error.  Removing for 1.0 is not enough time to allow clients to update.